crates/router/src/connector/stripe/transformers.rs

```diff
@@ -3638,6 +3674,18 @@ pub struct DisputeObj {
     pub status: String,
 }
 
+fn get_merchant_defined_metadata(metadata: Value) -> HashMap<String, String> {
```

Comment:
Since order_id is needed for all the flows, Please changes this method to accept order_id and construct the metadata with order_id here itself and return mutable hash map. If any other key/value is required in the flow then you can make changes in the flow itself

Reply:
Done